app/src/main/java/com/example/keybindhelper/dto/Project.java

```diff
@@ -36,7 +34,6 @@
      * to show if it belongs in room db or firebase
      */
     @Ignore
-    public boolean isFirebased=false;
```

Comment:
Removed this unused boolean after checking usages with 'Find Usages' feature.

Reply:
Good.